src/Command/Issue/IssueShowCommand.php

```diff
@@ -52,7 +54,18 @@ protected function configure()
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         if (null === $issueNumber = $input->getArgument('issue')) {
-            $issueNumber = $this->getHelper('git')->getIssueNumber();
```

Comment:
i am wondering why this does not work well... I am doing some reviews here https://github.com/sstok/gush/pull/8